lib/farmoney/money.rb

```diff
@@ -20,6 +20,14 @@ def to_s
       format("£%.1i.%.2i", pounds, pence)
     end
 
+    def attributes
+      instance_variables.each.with_object({}) do |attribute, hash|
```

Comment:
according to [THIS](https://apidock.com/rails/Enumerable/each_with_object), each_with_object is deprecated. Looks like it is a result of [THIS](https://bugs.ruby-lang.org/issues/10549) request 4 years ago. Backed up by [THIS](https://stackoverflow.com/questions/14671881/how-does-enumwith-object-differ-from-enumeach-with-object/14672305#14672305) SO post explaining that .with_object just takes an Enumerable, and should not have such a narrow scope of just `each`.